### ISSUE TITLE
error:Forbidden on text elements in Unyson. A host issue.

### ISSUE THREAD

**Issue by kierankay0** [ThemeFuse/Unyson#3479]
I have been getting the error message 'error:Forbidden' when using the divider, text group and text block elements in Unyson's visual page editor running on the wordpress theme Jevelin.

When one of these elements is clicked to be edited the popup window does not load and instead says error:Forbidden. After speaking to support of my page's hosting site JustHost, they found that the issue is that the plugin is incompatible with JustHost. However, yesterday the hosting site fixed the error briefly before it returned half a day later.

I, and my hosting site would like to know any solutions or modifications/updates that can be done at either end to allow full compatibility to remove this error as this is the only (but big) issue I have been faced with when using Unyson.

Cheers,
Kieran

![grab](https://user-images.githubusercontent.com/40665063/42036096-6c6c3c7c-7adc-11e8-8bcf-d903ea198186.jpeg)

**Comment by ViorelEremia** [comment]
Did your hosting say that is not compatible with; what we must change to be compatible?

**Comment by kierankay0** [comment]
Unfortunately they didn't say specifically what it was that was causing it to be incompatible. However the fix that worked for half a day involved them changing the php to 5.6. I wonder if this has anything to do with it and why the problem then came back later on?

**Comment by ViorelEremia** [comment]
Maybe you have some functions not compatibility with the newest version of php.
Can you send me some temporary credentials (admin, ftp): ```viorel@themefuse.com```
Please provide a link to this thread when submitting the message.

**Comment by kierankay0** [comment]
Thanks for getting back to me.
As of now it is working and I am no longer getting the error however this happened the other day and the error returned. My host site say they have whitelisted ModSecurity so it should work all the time now.

Do you know of any reason the error may occur again as if it does I’d like to know a quick solution and explanation to save me from running through all the problems again?

Thanks.

**Comment by ViorelEremia** [comment]
No, there is no way to avoid it. Only the server administrator can change it. He has special privileges because no one else besides him can make major changes on server so if you have this problem again then contact them we can not do anything to help you. but if it is disabled then the problem will not appear until one of the admins will activate it again.

**Comment by kierankay0** [comment]
Ah okay I understand.

From what my host was saying, the solution is they have to whitelist mod_security everytime an error occurs.
They also say that these errors occur due to corrupt updates. Surely you are not releasing updates that could cause errors?

**Comment by ViorelEremia** [comment]
What means "due to corrupt updates"?

**Comment by kierankay0** [comment]
That updates released by unyson can sometimes be corrupt and this can trigger the errors to occur, requiring mod_security to need to be whitelisted again. (I’m just repeating what my host told me).

**Comment by ViorelEremia** [comment]
I don't know that they mean but we do updates but not "sometimes be corrupt" I don't know what to say but for 150 000+ users unyson is working if in the future this will appear on your site again and it will be from unyson we will think what to do. And that means and from others plugins that can reactivate the modulerewrite so it is hard to follow when this happens and from whom.
